RevenueCat/Editor/RevenueCatPostInstall.cs

```diff
@@ -1,12 +1,12 @@
 
-#if UNITY_IOS
```

Comment:
Oh why this? `UNITY_EDITOR`

Reply:
Well, without this, I was getting a bunch of compilation issues when exporting Subtester as an iOS project.
<img width="703" alt="image" src="https://user-images.githubusercontent.com/808417/208294880-6cb1c686-a82d-4487-8d7f-df1a9181a45d.png">

It seemed like it was setting the correct flags and adding storekit correctly, so it seems to be fine. Do lmk if you think this could bring issues though!

Reply:
I think if the exported project is fine this is good

Reply:
let's add a note so the next maintainer doesn't wonder the same thing